docs/JLBP-14.md

```diff
@@ -16,10 +16,10 @@ dependency on google-http-java-client broke multiple customers'
 builds](https://github.com/GoogleCloudPlatform/appengine-gcs-client/issues/71)
 when only half the artifacts in the project were pushed one afternoon.
 
-Version ranges can be a security hole. A project that depends on a
-library with a version range picks up new versions of that dependency as they're
-pushed to Maven Central. This can be exploited by someone with release privileges
-to slip malicious code into projects without proper review. A variant
+Version ranges can be a security hole. A project that uses a range to specify
+the version of a dependency picks up new versions of that dependency when they're
```

Comment:
This doesn't sound right because a "range" is not "the version of a dependency", it is multiple allowable versions.

Reply:
makes sense. changed "specify" to "select"